### PR TITLE
refactor:  sbCard to template syntax

### PR DESCRIPTION
**Pull request by lisilinhart**
## Pull request type

Jira Link: - 

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## How to test this PR

Compare: 
* https://blok.ink/?path=/story/design-system-components-sbcard--default
* https://storyblok-design-system-git-refactor-ar-12-card-storyblok-com.vercel.app/?path=/story/design-system-components-sbcard--default

It's also good to check in the app:
`cd storyblok-design-system && git checkout refactor/ar-12-card`
Then start the server of the app

## What is the new behavior?

- it uses the template syntax
- 
-

### CODE REVIEW

File: src/components/Card/CardContent.vue [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="sb-card__content">
+    <slot></slot>
```
Comment (vcpablo) [inline src/components/Card/CardContent.vue:3]: Can we use `slot` as self-closing tags?

File: src/components/Card/CardContent.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <div v-if="isLoading" class="sb-card__loading">
+      <SbLoading size="x-large" color="primary" />
+    </div>
+  </div>
+</template>
+
+<script>
+import SbLoading from '../Loading'
```
Comment (vcpablo) [inline src/components/Card/CardContent.vue:11]: May we add one line here?

File: src/components/Card/CardContent.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export default {
+  name: 'SbCardContent',
+  components: {
+    SbLoading,
+  },
+  inject: ['cardContext'],
```
Comment (vcpablo) [inline src/components/Card/CardContent.vue:17]: Are there enough component levels that justifies using `provide/inject`?
Can't we just use `props` normally here?

**Review by ademarCardoso — APPROVED** [review]

File: src/components/Card/CardContent.vue (hunk 3 above)
Comment (lisilinhart) [reply src/components/Card/CardContent.vue:17]: Not really since it's injected through slots

**Pull request merged**